neqo-client/src/main.rs

```diff
@@ -503,7 +503,7 @@ fn client(
         Rc::new(RefCell::new(EmptyConnectionIdGenerator::new(0))),
         local_addr,
         remote_addr,
-        &CongestionControlAlgorithm::NewReno,
+        CongestionControlAlgorithm::NewReno,
```

Comment:
This seems unrelated.  Perhaps we need a rust 1.48 PR that does all the clippy cleanup.